kqueen_ui/utils/loggers.py

```diff
@@ -0,0 +1,30 @@
+import os
+import yaml
+import logging.config
+import logging
+
+
+def setup_logging(default_path='/code/kqueen_ui/utils/logger_config.yml', default_level=logging.INFO, env_key='LOG_CFG'):
```

Comment:
This path can't be hardcoded, absolute path /code is specific for container deployment

Reply:
yep, already work on it, thx